app/views/reports/regions/show.html.erb

```diff
@@ -325,134 +211,60 @@
     </div>
   </div>
   <div class="d-lg-flex w-lg-50 pl-lg-2">
-    <div id="cumulative-registrations" data-period="<%= @period.to_s %>" class="mt-8px mx-0px mb-16px pb-4px bg-white br-4px bs-small d-lg-flex fd-lg-column justify-lg-between h-lg-full w-lg-full mt-lg-0 pb-inside-avoid b-print-black w-print-16cm">
+    <div id="cumulative-registrations"
+         data-period="<%= @period.to_s %>"
+         class="mt-8px mx-0px mb-16px pb-4px bg-white br-4px bs-small d-lg-flex fd-lg-column
+                justify-lg-between h-lg-full w-lg-full mt-lg-0 pb-inside-avoid b-print-black w-print-16cm">
       <div class="pt-20px px-20px">
+        <div class="d-flex mb-8px">
         <% if current_admin.feature_enabled?(:report_with_exclusions) %>
-          <div class="d-flex flex-1 mb-8px">
+          <div class="d-flex flex-1">
             <h3 class="mb-0px mr-8px">
               Registrations
             </h3>
             <%= render "definition_tooltip",
                        definitions: { "Total registered patients" => t("registered_patients_copy.total_registered_patients", region_name: @region.name),
                                       "Monthly registered patients" => t("registered_patients_copy.monthly_registered_patients", region_name: @region.name) } %>
           </div>
+        <% else %>
+          <h3 class="mb-0px w-75">
+            Monthly and cumulative registrations
```

Comment:
@prabhanshuguptagit this is title should be removed, it's old and should always show "Registrations" if we're now showing registrations instead of assigned patients.

Reply:
This will get cleaned up in the feature flag code cleanup, will remove in a follow up PR.